tests/installation/bootloader_uefi.pm

```diff
@@ -44,11 +44,18 @@ use bootloader_setup;
 use registration;
 use utils;
 use version_utils qw(is_jeos is_microos is_opensuse is_sle is_selfinstall);
+use Utils::Backends qw(is_ipmi);
 
 # hint: press shift-f10 trice for highest debug level
 sub run {
     my ($self) = @_;
 
+    # Press key 't' to let grub2 boot menu show up in serial console
+    if (is_ipmi && is_selfinstall && get_var('IPXE_UEFI')) {
```

Comment:
```if (is_ipmi && check_var('DESKTOP', 'textmode') && get_var('IPXE_UEFI'))``` is better.

Reply:
There is no DESKTOP control for SelfInstall installation at all, I think.

Reply:
Where pressing t depends on terminal. There is no need to press t on gfxterm I think, so the if condition here does not represent suitable judgement.